ADR/0027-availability-probe-framework.md

```diff
@@ -0,0 +1,102 @@
+# 27. Availability Probe Framework
+
+* Date 2023-07-06
+
+## Status
+
+Accepted
+
+## Context
+
+As a product team, we want a composite system availability metric for both Production
+and Staging environments, so that we can make roadmap decisions with easy-to-understand
+feedback about the reliability of the system in mind.
+  
+This metric should tell us the overall up time of RHTAP with respect to services under
+the control of Red Hat.
+
+RHTAP is defined to be available at a given moment if all of its components are
```

Comment:
`s/RHTAP/AppStudio/g`

Reply:
I'm somewhat ashamed to say that I can't say why this should be AppStudio and not RHTAP. What's the reason this does not define RHTAP's availability?